Whiskey/Whiskey.psd1

```diff
@@ -99,6 +99,7 @@
                             'Publish-WhiskeyBuildMasterPackage',
                             'Publish-WhiskeyNuGetPackage',
                             'Publish-WhiskeyProGetUniversalPackage',
+                            'Publish-WhiskeyPSObject'

```

Comment:
`Publish-WhiskeyPSObject` is an private, internal function so shouldn't be exported. Remove this line.